### DISCUSSION TITLE
Command completion prompt support.

### DISCUSSION THREAD

**Discussion by BruceChenQAQ** [ORelio/Minecraft-Console-Client#2375] [General]
Related PullRequest: https://github.com/MCCTeam/Minecraft-Console-Client/pull/2358
* **Implement a simulated pop-up window to display command completion suggestions on the console.**
* Fixed editing and rendering issues for double-width East Asian characters.
* Fix crash when input or output is redirected.
  * You can use Ctrl+P to print the input and the position of the cursor when the output is redirected.
* Fix crash after changing window size.
  * Since the window size change triggers the terminal to rearrange the text, there is still the problem of input and text layout errors, for which no solution has been found.

If you have found something that is not working properly or you are getting results that are not as expected, please submit a bug to us.

<hr /><em>This discussion was created from the release <a href='https://github.com/MCCTeam/Minecraft-Console-Client/releases/tag/brigadier-dev-prerelease'>Command completion prompt support.</a>.</em>

**Comment by ReinforceZwei** [comment]
Is shows unknown command when typing `/<internal cmd> _help`. Is it normal?
![image](https://user-images.githubusercontent.com/39955851/206895166-4753b217-f304-48eb-99af-6249c4b7fd97.png)
_What I typed above: `/useitem _help`, `/help useitem`_

**Reply by BruceChenQAQ** [reply]
Confirmed as a bug, the redirected command needs a separate executable branch added. Working on fixing it.

**Reply by BruceChenQAQ** [reply]
It has been fixed and the files in the release have been updated. (Counter: 2)
